modules/Microsoft.VirtualMachineImages/imageTemplates/.test/common/deploy.test.bicep

```diff
@@ -29,23 +29,14 @@ resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
 
 module resourceGroupResources 'dependencies.bicep' = {
   scope: resourceGroup
-  name: '${uniqueString(deployment().name, location)}-paramNested'
+  name: '${uniqueString(resourceGroupName, location)}-paramNested'
   params: {
     managedIdentityName: 'dep-<<namePrefix>>-msi-${serviceShort}'
     sigImageDefinitionName: 'dep-<<namePrefix>>-imgd-${serviceShort}'
     galleryName: 'dep<<namePrefix>>sig${serviceShort}'
   }
 }
 
-resource msi_roleAssignment 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
```

Comment:
I take it the role assignment on the subscription scope is not required (anymore)? However, if that's the case - how can the image builder still create the 'random' resource group. In this test, the Staging-RG is empty...

Reply:
The Image Builder is a 1st party Enterprise Application and has this right by default when you register the resource provider.

Reply:
The role assignment is not needed at the subscription scope. When you register the VM Image Builder resource provider you are granting the 1st party enterprise application this right. https://learn.microsoft.com/en-us/azure/virtual-machines/image-builder-overview?tabs=azure-powershell#permissions

Reply:
In any case I'll want to look into this a bit deeper, as this may also simplify our 'Microsoft.Compute/disks/.test/import' test.